src/CLE.jl

```diff
@@ -87,16 +100,57 @@ Methods:
 * `voigt_moduli(at)`
 * `voigt_moduli(calc, at)`
 * `voigt_moduli(C)`
+* `voigt_moduli(C11, C12, C44)`
 """
 voigt_moduli(at::AbstractAtoms) = voigt_moduli(calculator(at), at)
 
 voigt_moduli(calc::AbstractCalculator, at::AbstractAtoms) =
    voigt_moduli(elastic_moduli(calc, at))
 
-const voigtinds = [1, 5, 9, 4, 7, 8]
+const voigtinds = [1, 5, 9, 8, 7, 4] # xx yy zz yz xz xy
 
 voigt_moduli(C::Array{T,4}) where {T} = reshape(C, 9, 9)[voigtinds, voigtinds]
 
+function voigt_moduli(C11::T, C12::T, C44::T) where {T}
+   z = zero(C11)
+   return [C11 C12 C12 z   z    z
+           C12 C11 C12 z   z    z
+           C12 C12 C11 z   z    z
+           z   z   z   C44 z    z
+           z   z   z   z   C44  z
+           z   z   z   z   z    C44]
+end
+
+"""
+Convert from 3×3 stress matrix σ to Voigt 6-vector
+"""
+voigt_stress(σ::AbstractMatrix) = (@assert size(σ) == (3,3);
+                                   reshape((σ + σ')/2, :)[voigtinds])
+voigt_stress(calc::AbstractCalculator, at::AbstractAtoms) = voigt_stress(stress(calc, at))
+voigt_stress(at::AbstractAtoms) = voigt_stress(stress(at))
+
+"""
+Convert from 3×3 strain matrix ϵ to Voigt 6-vector
+"""
+voigt_strain(ϵ::AbstractMatrix) = (@assert size(ϵ) == (3,3);
+                                   reshape(ϵ + ϵ' - Diagonal(ϵ), :)[voigtinds])
+
+# FIXME maybe there's a better name for 3x3 stress tensor than `cauchy_stress`?
```

Comment:
Up for debate: is there a better name for the 3x3 matrix forms of stress and strain, as opposed to the Voigt 6-vectors?